jest.config.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
 
   // A list of paths to directories that Jest should use to search for files in
   // roots: [
-  //   "<rootDir>"
+  //   "test"
```

Comment:
I think this is not needed, see next comment.

Reply:
sometimes stryker leaves a `.stryker-tmp/...` directory after running it (reproducible when hitting `Ctrl+C`), running jest after that will also run the tests in those directories.

Also it just speeds up jest since it doesn't need to search for tests anywhere else.